CHANGELOG.md

```diff
@@ -2849,7 +2857,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Uncategorized
 - Added the ability to restore accounts from seed words.
 
-[Unreleased]: https://github.com/MetaMask/metamask-extension/compare/v10.12.2...HEAD
+[Unreleased]: https://github.com/MetaMask/metamask-extension/compare/v10.12.4...HEAD
+[10.12.4]: https://github.com/MetaMask/metamask-extension/compare/v10.12.3...v10.12.4
+[10.12.3]: https://github.com/MetaMask/metamask-extension/compare/v10.12.2...v10.12.3
 [10.12.2]: https://github.com/MetaMask/metamask-extension/compare/v10.12.1...v10.12.2
 [10.12.1]: https://github.com/MetaMask/metamask-extension/compare/v10.12.0...v10.12.1
```

Comment:
This would best reflect the real contents of these releases:
```suggestion
[10.12.4]: https://github.com/MetaMask/metamask-extension/compare/v10.12.2...v10.12.4
[10.12.3]: https://github.com/MetaMask/metamask-extension/compare/v10.11.2...v10.11.3
[10.12.2]: https://github.com/MetaMask/metamask-extension/compare/v10.12.0...v10.12.2
[10.12.1]: https://github.com/MetaMask/metamask-extension/compare/v10.11.3...v10.11.4
```

Reply:
@Gudahtt this change seems to cause the linter to fail

Reply:
I see, this is my vault, the validator wasn't written to account for rollback releases. It will have to do as-is then.